Sources/Kommunicate/Classes/CustomPreChatFormViewController.swift

```diff
@@ -7,8 +7,18 @@
 
 import UIKit
 
+class optionsCell : UITableViewCell {
```

Comment:
See if we already have an empty cell that we can use. Also, use Pascal case for types.

Reply:
we don't have empty cell

Reply:
This is pending: 
> Also, use Pascal case for types.